README.md

```diff
@@ -70,3 +73,4 @@ Enjoy!
 [11]: https://symfony.com/doc/3.4/logging.html
 [13]: https://symfony.com/doc/current/bundles/SensioGeneratorBundle/index.html
 [14]: https://symfony.com/doc/current/setup/built_in_web_server.html
+[15]: https://github.com/symfony/skeleton
```

Comment:
Probably should be pointed to https://github.com/symfony/website-skeleton instead? Looks like it will be a replacement for Symfony SE

Reply:
Hmm. Or maybe we should link to the Setup page on the docs (and there we can explain the skeleton options). Because if they’re on this repo... they made some mistake and need to start fresh with new info :)

Reply:
Setup page will give more context 👍